### PR TITLE
add footer code

### LINKED ISSUE #8: Footer Section
# Task 
* create footer section 
* style this section 
* make the section responsive

### PR DESCRIPTION
**Pull request by emanjaber**
Relates #8

These are the codes of the footer (html, style, media)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -12,3 +12,57 @@ font-family: 'Raleway', sans-serif;
 *{
     font-family: 'Raleway', sans-serif;
 }
+
+
+/* start footer */
+
+footer{
+    padding: 25px 0px;
```
Comment (Heba-elLouh) [inline css/style.css:210]: I think it needs height: 150px
and if you change it then you have to change the padding to 50px 0px;

**Review by Heba-elLouh — DISMISSED** [review]

**Review by sara219 — CHANGES_REQUESTED** [review]
resolve the conflict

File: css/style.css (hunk 1 above)
Comment (emanjaber) [reply css/style.css:210]: Thanks for your suggestion, I tried it but found the height is just too big compared to the content. On the other hand, I have taken care that the top and bottom margins are not equal, so thank you again!

**Comment by emanjaber** [comment]
I resolved the conflict @sara219 , but I think a new one will appear again when another branch is merged before it.

**Review by reemsh2000 — DISMISSED** [review]

**Review by Heba-elLouh — DISMISSED** [review]

**Comment by emanjaber** [comment]
I resolved the conflict again!😭😂 See the code, please👉👈😗@sara219

**Review by sara219 — APPROVED** [review]

**Review by reemsh2000 — APPROVED** [review]

**Review by Heba-elLouh — APPROVED** [review]

**Pull request merged**